### PR TITLE
BaSP-TG-68: Task List

### PR DESCRIPTION
**Pull request by AlejandroSantini**

### CODE REVIEW

File: src/Components/Tasks/index.js [hunk 1]
```diff
@@ -1,44 +1,97 @@
...
+  const [searchQuery, setSearchQuery] = useState('');
+  const [untouchedData, setUntouchedData] = useState([]);
 
   useEffect(async () => {
     try {
+      setLoading(true);
       const response = await fetch(`${process.env.REACT_APP_API_URL}/tasks`);
       const responseJson = await response.json();
-      setList(responseJson.data);
+      console.log(responseJson.data);
```
Comment (laura-brussa12) [inline src/Components/Tasks/index.js:22]: Maybe you should delete this console.log

File: src/Components/Layout/layout.module.css [hunk 2]
```diff
@@ -15,6 +15,7 @@
 }
 
 .divSwitch{
+  margin-top: 6%;
```
Comment (laura-brussa12) [inline src/Components/Layout/layout.module.css:18]: Why you add this?

File: src/Components/Tasks/index.js [hunk 3]
```diff
@@ -1,44 +1,97 @@
...
+      setDataTable(result);
+      setIsOpen(false);
+    } catch (error) {
+      console.log(error);
+    } finally {
+      setLoading(false);
     }
   };
 
+  const column = [
```
Comment (laura-brussa12) [inline src/Components/Tasks/index.js:59]: Maybe you should move this const to line 16 in order to have all the const together

File: src/Components/Tasks/index.js [hunk 4]
```diff
@@ -1,44 +1,95 @@
...
+      <Modal modalTitle={'Delete Task'} isOpen={isOpen} handleClose={() => setIsOpen(!isOpen)}>
+        <p>Do you wish to eliminate the task?</p>
+        <div>
+          <Button text="DELETE" type="delete" handler={() => deleteItem(deleteId)} />
+          <Button text="CANCEL" handler={() => setIsOpen(!isOpen)} />
+        </div>
+      </Modal>
+      <h2 className={styles.h2Task}>Tasks</h2>
+      <div className={styles.search}>
+        <Search searchQuery={searchQuery} setSearchQuery={search} placeholder="Search by name" />
```
Comment (GuidoCerioni) [inline src/Components/Tasks/index.js:86]: Placeholder should be "Search by id"

File: src/Components/Tasks/index.js [hunk 5]
```diff
@@ -1,44 +1,95 @@
...
+      </div>
+      <div className={styles.btnTask}>
```
Comment (GuidoCerioni) [inline src/Components/Tasks/index.js:88]: Search and Button 'Add Task' are disalign. You can put them in the same container
image

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
Check comments!

**Review by GuidoCerioni — DISMISSED** [review]
Weel done!!

**Review by Gearyandres94 — DISMISSED** [review]
Great job!!

**Review by HiginiaMed — APPROVED** [review]
Good job Ale!!

**Review by Gearyandres94 — APPROVED** [review]

**Review by GuidoCerioni — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]

**Pull request merged**